tests/tiling/tools/collect_mesh_sizes.py

```diff
@@ -0,0 +1,71 @@
+from firedrake import *
+
+from os import listdir, environ
+from os.path import isfile, join
+import sys
+import mpi4py
+
+
+def get_S_U_dofs(mesh, p):
```

Comment:
This is really useful. The same technique to get the S DoF count in `seigen/elastic.py` (see `ElasticLF4` class). Perhaps move this higher up to `seigen/helpers.py`?